### DISCUSSION TITLE
How to build 2sxc for Oqtane

### DISCUSSION THREAD

**Discussion by SSzretter** [2sic/2sxc#3462] [General]
Hello, I cloned and successfully built the 2sxc multi target solution but it looks like the ToSic.Sxc.Oqt.Package project never calls the debug.cmd script that copies the various files into the Oqtane.Server project.   Wild guess, it is one of the conditions such as  $(ConfigurationName)|$(OqtaneExist)  or  $(CurrentOqtaneRoot) but it is not clear to me where these are coming from.   Can I please have a hint on what I need to do to target Oqtane?

**Comment by iJungleboy** [comment]
Thanks for asking.

First please make sure you have the setup as described here.
https://docs.2sxc.org/abyss/contribute/code.html

Then for Oqtane (not perfect ATM) you will need to run the package project to make it build/deploy to your dev oqtane.

![image](https://github.com/user-attachments/assets/bff781ba-3bbe-4855-b50c-21c0e8d4ecce)
